docs/user/ml/index.asciidoc

```diff
@@ -223,3 +223,9 @@ point type selector to filter the results by specific types of change points.
 
 [role="screenshot"]
 image::user/ml/images/ml-change-point-detection-selected.png[Selected change points]
+
+You can attach selected charts to a dashboard or a case by using the context 
+menu. You can either preserve the applied time range or use the time bound from 
+the page date picker, configure a specific partition, or set the maximum number 
```

Comment:
in my previous comments, I mentioned `specific partitions`. The plural is important here, because the user can select multiple charts if the split field is selected.

Reply:
Please check the comment below